sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample7_CopyCustomModel.md

```diff
@@ -50,7 +50,7 @@ CopyAuthorization targetAuth = await targetClient.GetCopyAuthorizationAsync(reso
 string jsonTargetAuth = targetAuth.ToJson();
 ```
 
-To deserealize a string that contains authorizaiton information, use the `FromJson` method from `CopyAuthorization`.
```

Comment:
I'm sure this was me. I always have a hard time spelling `deserialize`.
Thanks for fixing this!